src/components/FeatureViews/Scheduler/EditSessionView.js

```diff
@@ -139,19 +136,19 @@ function EditSessionView({ course, session, editSelection }) {
         setSessionFields({
             ...sessionFields,
             duration: event.target.value,
-
+            tempDuration: newEndTime
```

Comment:
update "end_time" not this "tempDuration", this is too confusing

Reply:
see above